src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -37,20 +36,17 @@ public interface IChartEditorPresenter : IPresenter<IChartEditorView>,
       void Clear();
 
       /// <summary>
-      ///    Adds all columns of <paramref name="dataRepository" /> to DataBrowser. This  does not trigger a redraw of the chart
-      /// </summary>
-      void AddDataRepository(DataRepository dataRepository);
-
-      /// <summary>
-      ///    Removes all columns of <paramref name="dataRepository" /> from DataBrowser. This does not trigger a redraw of the
+      ///    Removes all columns of all <paramref name="dataRepositories" /> from DataBrowser. This does not trigger a redraw of
+      ///    the
       ///    chart
       /// </summary>
-      void RemoveDataRepository(DataRepository dataRepository);
+      void RemoveDataRepositories(IEnumerable<DataRepository> dataRepositories);
 
       /// <summary>
-      ///    Remove unused columns (e.g. not attached to a <see cref="DataRepository"/>) and adds columns from <paramref name="dataRepository"/>. This does not trigger a redraw of the chart
+      ///    Remove unused columns (e.g. not attached to a <see cref="DataRepository">.</see>This does not trigger a redraw of
+      ///    the chart
       /// </summary>
-      void RemoveUnusedColumnsAndAdd(DataRepository dataRepository);
```

Comment:
This was a weird method that was adding stuff and removing at the same time. I tried to rename it but still it feels like it should just removed unused columns. if someone wants to add columns after that, then let;s calll the addMethod